includes/integrations/sprout-invoices/class-integration-sprout-invoices.php

```diff
@@ -83,6 +83,15 @@ public function create_doc($entry_id, $form_id, $page_id)
         if (false === $form_data) {
             return;
         }
+
+        // $form_payment_data = weforms_get_form_payments( $form_id );
+
+        // error_log("Form Payment Data" . " = " . print_r($form_payment_data,1));
+
+        $payment_data = weforms_get_entry_payment( $entry_id );
+
+        error_log( $payment_data );
```

Comment:
is this error_log supposed to be here?

Reply:
should have been removed. Will remove

Reply:
weforms_get_entry_payment isnt going to be used at this time as well